From d6a796c3e54a9f4199ee275da6980880e0fa19a6 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 15 Jun 2019 18:44:19 +0200
Subject: [PATCH] db8: Remove references to
 com.webos.service.attachedstoragemanager

We don't have PDM (com.webos.service.attachedstoragemanager) available in LuneOS (yet), therefore remove it for now, so it doesn't give errors in logs such as:

Jun 15 11:42:28 qemux86-64 user.err ls-hubd: [] [pmlog] ls-hubd LSHUB_NOT_LSTED {"SERVICE_NAME":"com.webos.service.attachedstoragemanager","EXE":"/usr/sbin/mojodb-luna","APP_ID":"(null)","PID":453} Service not listed in service files (cmdline: /usr/sbin/mojodb-luna -c /e

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 files/sysbus/dynamic/com.palm.db.role.json.in              | 6 +++---
 files/sysbus/dynamic/com.webos.db8.test.media.role.json.in | 2 +-
 files/sysbus/static/com.palm.db.role.json.in               | 6 +++---
 files/sysbus/static/com.webos.db8.test.media.role.json.in  | 2 +-
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/files/sysbus/dynamic/com.palm.db.role.json.in b/files/sysbus/dynamic/com.palm.db.role.json.in
index 9e65a14..6324127 100644
--- a/files/sysbus/dynamic/com.palm.db.role.json.in
+++ b/files/sysbus/dynamic/com.palm.db.role.json.in
@@ -5,15 +5,15 @@
     "permissions": [
         {
             "service":"com.palm.db",
-            "outbound":["com.webos.service.activitymanager.client","com.palm.service.backup","com.palm.systemmanager","com.palm.activitymanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice"]
+            "outbound":["com.webos.service.activitymanager.client","com.palm.service.backup","com.palm.systemmanager","com.palm.activitymanager", "com.webos.settingsservice"]
         },
         {
             "service":"com.palm.tempdb",
-            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice"]
+            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.settingsservice"]
         },
         {
             "service":"com.webos.mediadb",
-            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice"]
+            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.settingsservice"]
         }
     ]
 }
diff --git a/files/sysbus/dynamic/com.webos.db8.test.media.role.json.in b/files/sysbus/dynamic/com.webos.db8.test.media.role.json.in
index a22a525..c0154a5 100644
--- a/files/sysbus/dynamic/com.webos.db8.test.media.role.json.in
+++ b/files/sysbus/dynamic/com.webos.db8.test.media.role.json.in
@@ -5,7 +5,7 @@
     "permissions": [
         {
             "service":"com.webos.db8.test.media",
-            "outbound":["com.webos.service.attachedstoragemanager"]
+            "outbound":["*"]
         }
     ]
 }
diff --git a/files/sysbus/static/com.palm.db.role.json.in b/files/sysbus/static/com.palm.db.role.json.in
index 9e65a14..6324127 100644
--- a/files/sysbus/static/com.palm.db.role.json.in
+++ b/files/sysbus/static/com.palm.db.role.json.in
@@ -5,15 +5,15 @@
     "permissions": [
         {
             "service":"com.palm.db",
-            "outbound":["com.webos.service.activitymanager.client","com.palm.service.backup","com.palm.systemmanager","com.palm.activitymanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice"]
+            "outbound":["com.webos.service.activitymanager.client","com.palm.service.backup","com.palm.systemmanager","com.palm.activitymanager", "com.webos.settingsservice"]
         },
         {
             "service":"com.palm.tempdb",
-            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice"]
+            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.settingsservice"]
         },
         {
             "service":"com.webos.mediadb",
-            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice"]
+            "outbound":["com.webos.service.activitymanager.client","com.palm.activitymanager","com.palm.systemmanager", "com.webos.settingsservice"]
         }
     ]
 }
diff --git a/files/sysbus/static/com.webos.db8.test.media.role.json.in b/files/sysbus/static/com.webos.db8.test.media.role.json.in
index a22a525..c0154a5 100644
--- a/files/sysbus/static/com.webos.db8.test.media.role.json.in
+++ b/files/sysbus/static/com.webos.db8.test.media.role.json.in
@@ -5,7 +5,7 @@
     "permissions": [
         {
             "service":"com.webos.db8.test.media",
-            "outbound":["com.webos.service.attachedstoragemanager"]
+            "outbound":["*"]
         }
     ]
 }
-- 
2.17.1.windows.1

